src/js/components/jtable/TableRow.jsx

```diff
@@ -13,6 +14,7 @@ type Props = {
     className?: string,
     children?: ReactChildren,
     href?: string,
+    linkTo?: string,
```

Comment:
I don't want anything wrapping the rows and cells, so the Link has to be the row. And JDL is already dependent on react-router for other stuff.